### DISCUSSION TITLE
Are there flags for Parallelism?

### DISCUSSION THREAD

**Discussion by ollie-kane-CB** [slingdata-io/sling-cli#352] [Q&A]
Is there a way to execute Streams of copying in parallel?  My usecase could benefit from increased performance provided I could write using multiple streams at once.

I looked for similar topics and didn't see any, which I found surprising.  Have I asked the wrong question perhaps and this behavior is by design as there's another preferred method?

**Comment by flarco** [comment]
See [here](https://docs.slingdata.io/sling-cli/cli-pro#parallel-processing-and-retries).
